### PR TITLE
RELEASE 1.28.0: Added FlashBlade REST 2.5 and 2.6

### PR DESCRIPTION
**Pull request by tlewis-ps**
New SDK Features:

    The "Product Type", which reflects the product line and configuration of the array, has been added.
    See: get_arrays()

    Exposed the eradication settings for data on the array.
    See: get_arrays()

    Added the ability to run ping commands from specific components of the array.
    See: get_network_interfaces_ping()

    Added the ability to perform traceroute commands from specific components of the array.
    See: get_network_interfaces_trace()

    Exposed a new Retention Lock setting for snapshot policies.
    See: get_file_system_snapshots()

### CODE REVIEW

File: pypureclient/flashblade/FB_2_4/models/nfs_export_policy_post.py [hunk 1]
```diff
@@ -36,8 +36,7 @@ class NfsExportPolicyPost(object):
         'is_local': 'bool',
         'location': 'FixedReference',
         'policy_type': 'str',
-        'rules': 'list[NfsExportPolicyRuleInPolicy]',
-        'version': 'str'
```
Comment (tlewis-ps) [inline pypureclient/flashblade/FB_2_4/models/nfs_export_policy_post.py:40]: No clue how `version` snuck in here in our last release, but it's readonly. Anyone who happens to be setting it in a POST for some reason is already failing. Removing it shouldn't cause any issues.

File: docs/changelog.rst [hunk 2]
```diff
@@ -6,6 +6,7 @@ A log of changes by version and date.
 ======= ========== =====
 Version Date       Notes
 ======= ========== =====
+1.28.0  09/21/2022 Added FlashBlade 2.5 and 2.6 clients
```
Comment (tlewis-ps) [inline docs/changelog.rst:9]: Note: Manually updated this to match our actual GA date. I'll be updating this in the internal repo as well.

**Review by ashahid-ps — COMMENTED** [review]
We should add the example files for network ping and traceroute?

**Comment by tlewis-ps** [comment]
> We should add the example files for network ping and traceroute?

Done, @ashahid-ps  👍

**Review by ashahid-ps — APPROVED** [review]

**Pull request merged**